sqlx-postgres/src/types/ltree.rs

```diff
@@ -95,11 +95,6 @@ impl PgLTree {
         Self::default()
     }
 
-    /// creates ltree from a [`Vec<PgLTreeLabel>`]
-    pub fn from(labels: Vec<PgLTreeLabel>) -> Self {
```

Comment:
Let's keep this but just rename it to `from_labels()`. That way there's an alternative for contexts where the compiler might have trouble inferring the correct `From` impl.